local/mymedia/mymedia.php

```diff
@@ -54,8 +54,6 @@
     'height' => '600px',
     'width' => '100%',
     'allowfullscreen' => 'true',
-    'webkitallowfullscreen' => 'true',
```

Comment:
I know that these are basically deprecated and not needed, but any other reason for removing them?
are we sure this won't cause regression in some weird old browsers which will make us return them back per a customer's request in couple of weeks/months?